src/domain/prices/prices.repository.ts

```diff
@@ -37,9 +37,8 @@ export class PricesRepository implements IPricesRepository {
       tokenAddress: lowerCaseTokenAddress,
       fiatCode: lowerCaseFiatCode,
     });
-    const assetPrice = this.assetPriceValidator.validate(result);
```

Comment:
Could this be an issue also in `getNativeCoinPrice`?

Reply:
For native coins, the response is always `{}` when not found. In those cases the validation will throw an error an we will log it and map its price to `null` at the routes level. 

A not-found native coming price shouldn't ever happen, but anyway, it doesn't apply because if it happens we are not extending the TTL as for the ERC20 tokens.